test/thorest-api/debug/tracers.test.ts

```diff
@@ -50,13 +50,10 @@ describe('POST /debug/tracers', () => {
             config,
         })
 
-    it.e2eTest('should return 200 for no tracer', 'all', async () => {
+    it.e2eTest('should return 403 for no tracer', 'all', async () => {
         const res = await newRequest('')
 
-        expect(res.httpCode).toBe(200)
-        verifyStructLogs(res.body.structLogs)
```

Comment:
not sure if this is needed, as linter finds it not used anywhere ^

Reply:
I dont have that line 🤔 
<img width="912" alt="image" src="https://github.com/user-attachments/assets/f42cb832-0781-4bbd-9ca5-1d8c1f4328d2">

Reply:
Ah, nvm, found it 👍